Apply meeting feedback (#2)
transmute-industries/ietf-spice-charter: charter.md

    - https://datatracker.ietf.org/doc/rfc9278/
    - https://datatracker.ietf.org/doc/draft-ietf-cose-key-thumbprint/

> Based on conversations on OAUTH and SPICE lists, the following work items have been proposed to be moved to SPICE, assuming authors / WGs agree, and SPICE forms to be able to receive them:

Reviewer: ```suggestion
Based on conversations on OAUTH and SPICE lists, the following work items are potential candidates to be moved to SPICE, assuming authors / WGs agree, and SPICE forms to be able to receive them:
```

## Why IETF

> Because of the security and integrity aspects directly involved in verifiable credentials that utilize a Three Role Model, particularly their reliance on asymmetric cryptography with possible intermediaries during exchange, this topic must be worked with in cooperation with existing security expertise in the IETF and IRTF.  The broader community engaging in the verifiable credential space is converging within the IETF because of this security expertise. Multiple editors of existing work in W3C related to verifiable credentials are engaged and supportive of this work, and believe that IETF is the appropriate location for this work. By working in IETF we can ensure broader engagement with the work, security, and privacy review by the appropriate experts at IETF, and better coordinate with IETF groups that are leveraging or extending this work for use in various IETF standards.

Reviewer: ```suggestion
Because of the security and integrity aspects directly involved in verifiable credentials that utilize a "Three Role Model", particularly their reliance on asymmetric cryptography with possible intermediaries during exchange, this topic must be worked with in cooperation with existing security expertise in the IETF and IRTF.  The broader community engaging in the verifiable credential space is converging within the IETF because of this security expertise. Multiple editors of existing work in W3C related to verifiable credentials are engaged and supportive of this work, and believe that IETF is the appropriate location for this work. By working in IETF we can ensure broader engagement with the work, security, and privacy review by the appropriate experts at IETF, and better coordinate with IETF groups that are leveraging or extending this work for use in various IETF standards.
```
